cmd/execute-cadence-script/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/onflow/cadence"
+	"github.com/onflow/flow-go-sdk/client"
+
+	"github.com/rs/zerolog"
+	"github.com/spf13/pflag"
+	"google.golang.org/grpc"
+)
+
+func main() {
+
+	var (
+		flagAPI        string
+		flagScriptFile string
+		flagLogLevel   string
+		flagHeight     int64
+	)
+
+	// TODO: mismatch between block height type - using int64 instead of uint64
+
+	pflag.StringVarP(&flagScriptFile, "script-file", "s", "", "cadence script to execute")
+	pflag.StringVarP(&flagAPI, "api", "a", "localhost:3569", "access node API address")
+	pflag.StringVarP(&flagLogLevel, "log-level", "l", "info", "log level for JSON logger")
+	pflag.Int64VarP(&flagHeight, "height", "h", -1, "height on which to execute script")
+
+	pflag.Parse()
+
+	zerolog.TimestampFunc = func() time.Time { return time.Now() }
+	log := zerolog.New(os.Stderr).With().Timestamp().Logger().Level(zerolog.DebugLevel)
+	level, err := zerolog.ParseLevel(flagLogLevel)
+	if err != nil {
+		log.Fatal().Err(err)
```

Comment:
This won't work, it just creates Fatal-level event with an error attached that is never sent.

Do the following if you wish to just send it without a message

```suggestion
		log.Fatal().Err(err).Send()
```

Or attach a message

```suggestion
		log.Fatal().Err(err).Str("log_level_flag", flagLogLevel).Msg("could not parse log level flag")
```

I'd also suggest adding the actual value that causes an issue as an attribute of the log. (See example above)

Reply:
Nice catch, that was a very pretty no-op :D